apis/run/v1alpha2/zz_generated.conversion.go

```diff
@@ -0,0 +1,180 @@
+	}); err != nil {
+		return err
+	}
+	if err := s.AddGeneratedConversionFunc((*v1alpha3.TanzuKubernetesRelease)(nil), (*TanzuKubernetesRelease)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha3_TanzuKubernetesRelease_To_v1alpha1_TanzuKubernetesRelease(a.(*v1alpha3.TanzuKubernetesRelease), b.(*TanzuKubernetesRelease), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddGeneratedConversionFunc((*TanzuKubernetesReleaseList)(nil), (*v1alpha3.TanzuKubernetesReleaseList)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha1_TanzuKubernetesReleaseList_To_v1alpha3_TanzuKubernetesReleaseList(a.(*TanzuKubernetesReleaseList), b.(*v1alpha3.TanzuKubernetesReleaseList), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddGeneratedConversionFunc((*v1alpha3.TanzuKubernetesReleaseList)(nil), (*TanzuKubernetesReleaseList)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha3_TanzuKubernetesReleaseList_To_v1alpha1_TanzuKubernetesReleaseList(a.(*v1alpha3.TanzuKubernetesReleaseList), b.(*TanzuKubernetesReleaseList), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddGeneratedConversionFunc((*TanzuKubernetesReleaseStatus)(nil), (*v1alpha3.TanzuKubernetesReleaseStatus)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha1_TanzuKubernetesReleaseStatus_To_v1alpha3_TanzuKubernetesReleaseStatus(a.(*TanzuKubernetesReleaseStatus), b.(*v1alpha3.TanzuKubernetesReleaseStatus), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddGeneratedConversionFunc((*v1alpha3.TanzuKubernetesReleaseStatus)(nil), (*TanzuKubernetesReleaseStatus)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha3_TanzuKubernetesReleaseStatus_To_v1alpha1_TanzuKubernetesReleaseStatus(a.(*v1alpha3.TanzuKubernetesReleaseStatus), b.(*TanzuKubernetesReleaseStatus), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddConversionFunc((*TanzuKubernetesReleaseSpec)(nil), (*v1alpha3.TanzuKubernetesReleaseSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha1_TanzuKubernetesReleaseSpec_To_v1alpha3_TanzuKubernetesReleaseSpec(a.(*TanzuKubernetesReleaseSpec), b.(*v1alpha3.TanzuKubernetesReleaseSpec), scope)
+	}); err != nil {
+		return err
+	}
+	if err := s.AddConversionFunc((*v1alpha3.TanzuKubernetesReleaseSpec)(nil), (*TanzuKubernetesReleaseSpec)(nil), func(a, b interface{}, scope conversion.Scope) error {
+		return Convert_v1alpha3_TanzuKubernetesReleaseSpec_To_v1alpha1_TanzuKubernetesReleaseSpec(a.(*v1alpha3.TanzuKubernetesReleaseSpec), b.(*TanzuKubernetesReleaseSpec), scope)
+	}); err != nil {
+		return err
+	}
+	return nil
+}
+
+func autoConvert_v1alpha1_TanzuKubernetesRelease_To_v1alpha3_TanzuKubernetesRelease(in *TanzuKubernetesRelease, out *v1alpha3.TanzuKubernetesRelease, s conversion.Scope) error {
```

Comment:
These need to be regenerated, and the function names and stuff might change. It might be better to leave the conversion code out of the first part and add that in later in a separate PR or a separate commit in the same PR.